### PR TITLE
[5.x] Improve addons listing

### PR DESCRIPTION
**Pull request by duncanmcclean**
This pull request makes various UI improvements to the "Addons" page in the Control Panel. It's still not perfect, but it's way better than what we have right now.

## Screenshots

CleanShot 2024-09-18 at 15 23 19

CleanShot 2024-09-18 at 15 23 46

### CODE REVIEW

File: resources/views/addons/index.blade.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 @extends('statamic::layout')
 @section('title', __('Addons'))
+@section('wrapper_class', 'max-w-full')
```
Comment (jasonvarga) [inline resources/views/addons/index.blade.php:5]: I think there should still be a max width. Wider than it was, but not full. It's ok on the listing but on the show page it's awkward.

Comment (duncanmcclean) [reply resources/views/addons/index.blade.php]: Oh, good catch! My browser was only 1/2 of my screen but looking at it now, it's silly. 🙈

Any better now?

Comment (jasonvarga) [reply resources/views/addons/index.blade.php]: Yup

**Review by jasonvarga — CHANGES_REQUESTED** [review]
When on a narrow viewport, the description is now above the meta data and is missing margin.

CleanShot 2024-09-18 at 11 24 49

**Comment by duncanmcclean** [comment]
@jasonvarga Fixed!

**Comment by jasonvarga** [comment]
You only fixed the spacing. The description is still first. The meta/install cards should go first.

**Comment by duncanmcclean** [comment]
> You only fixed the spacing. (Which is great 👌 ) The description is still first. The meta/install cards should go first.

Ohhh.... that should be fixed now too 😅

**Review by jasonvarga — APPROVED** [review]
Looks good to me.

**Pull request merged**